README.rst

```diff
@@ -3,7 +3,7 @@ nb.lsst.io documentation
 ########################
 
 This is the (prototype) documentation for the notebook aspect of the LSST Science Platform.
-This site supports our current users of https://lsst-lspdev.ncsa.illinois.edu/nb and is proof-of-concept for documenting the LSST Science Platform as a whole.
+This site supports our current users of https://lsst-lsp-stable.ncsa.illinois.edu/ and is a proof-of-concept for documenting the LSST Science Platform as a whole.
```

Comment:
Here and below, we still need the `/nb` extension in order to go directly to the notebook login page.

Reply:
Given that this is documentation specifically for the notebook aspect, I'd advocate for keeping `nb` in the exposed URLs.